tests/e2e/blocks/audio-video.spec.js

```diff
@@ -0,0 +1,71 @@
+import {test, expect} from '../tools/lib/test-utils.js';
+import {publishPostAndVisit, createPostWithFeaturedImage} from '../tools/lib/post.js';
+import {searchAndInsertBlock} from '../tools/lib/editor.js';
+
+const YOUTUBE_TEST = 'https://www.youtube.com/watch?v=-CwkccAgKrs';
+const VIMEO_TEST = 'https://vimeo.com/100195272';
+const MP4_TEST = 'https://www.greenpeace.org/static/planet4-assets/tests/edge_of_the_wrold.mp4';
+
+const SOUNDCLOUD_TEST = 'https://soundcloud.com/greenpeaceuk-1/04-and-we-will-defend-requiem';
+const MP3_TEST = 'https://www.greenpeace.org/static/planet4-assets/tests/wochenserie_greenpeace.mp3';
+
+/**
+ * Add a Video or Audio block to a page with a specific link.
+ *
+ * @param {Object} page - The page object for interacting with the browser.
+ * @param {string} mediaLink - The video file link (can be YouTube, Vimeo, mp4).
+ * @param {string} mediaType - The type of media added (Audio or Video).
+ */
+const addVideoOrAudioBlock = async ({page}, mediaLink, mediaType) => {
```

Comment:
This is a small change but just for a preference I'd like to set the `type` first, and the the `link` which is extremely related to its type.

Reply:
Sure, I don't have strong feelings either way